Packs/DeveloperTools/Scripts/VerifyContextFieldsList/VerifyContextFieldsList.yml

```diff
@@ -0,0 +1,30 @@
+args:
+- name: fields_to_search
+  required: true
+  description: Fields to search if there are exsist in map
+  isArray: true
+- name: context
+  required: true
+  description: Map to search fields in
+  isArray: true
+comment: Receive fields and context. Verifies that these field exists in context.
+commonfields:
+  id: VerifyContextFieldsList
+  version: -1
+enabled: false
+name: VerifyContextFieldsList
```

Comment:
I think the name should be VerifyObjectFieldsList as the script alone has no relation to context.